package.json

```diff
@@ -1,8 +1,8 @@
 {
     "private": true,
     "scripts": {
-        "dev": "turbo run dev --filter=./apps/wfo-ui",
-        "dev:surf": "turbo run dev --filter=./apps/wfo-ui-surf",
+        "dev": "turbo run dev --filter=./apps/wfo-ui --filter=./packages/*",
```

Comment:
is the "./" needed?

I think is also works like this:
--filter=packages/*

Reply:
Does this run all packages in packages? Is that what we want?

Reply:
In this branch/PR yes, when the PR of Georgi gets involved we have to prevent the surf-package will run when we start local dev with the `dev` command

Reply:
According to one of the examples in the docs, the ./ needs to be there: https://turbo.build/repo/docs/core-concepts/monorepos/filtering#filter-by-directory